metricbeat/module/elasticsearch/node_stats/_meta/data.json

```diff
@@ -1,105 +1,118 @@
 {
     "@timestamp": "2016-05-23T08:05:34.853Z",
```

Comment:
Is this file is a test output? Why isn't the timestamp changed here? I am asking because I would like to understand this PR fully.

Reply:
Good question. I would also expect it to change. Let me check.

There is an additional issue here. @urso recently change the generation of the to our internal json library where an beat.Event can be passed. Unfortunately this has the side affect, that it also adds @metadata which I think is confusing and I remove it manually below. Also it changes indentation from 4 to 2 which changes all the files.

I would prefer to decouple the above issues from this PR that is why I did some manual tweaking.